src/components/com_tjcertificate/administrator/models/certificate.php

```diff
@@ -188,4 +188,93 @@ public function getCertificateProviderInfo($contentId, $client)
 
 		return trim(implode("\n", $html));
 	}
+
+	/**
+	 * Method to delete record
+	 *
+	 * @param   int  $certificateId  post data
+	 *
+	 * @return	JForm	A JForm object on success, false on failure
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function delete($certificateId)
+	{
+		$table    = $this->getTable('Certificates');
+		$table->load(array('id' => (int) $certificateId));
+
+		if ($table->delete($table->id))
+		{
+			JLoader::import('components.com_tjcertificate.events.record', JPATH_SITE);
+			$tjCertificateTriggerRecord = new TjCertificateTriggerRecord;
+
+			$tjCertificateTriggerRecord->onAfterRecordDeleted($table);
+			$dispatcher = \JEventDispatcher::getInstance();
+			$dispatcher->trigger('onExternalCertificateAfterDelete', array($table));
+		}
+	}
```

Comment:
is it getting called for normal certificates as well when we delete from admin side?

Reply:
function updated